test/postgresql.test.js

```diff
@@ -263,6 +263,18 @@ describe('postgresql connector', function() {
       });
   });
 
+  it('should support where filter for array type field', function(done) {
+    Post.dataSource.settings.allowExtendedOperators = true;
+    Post.find({where: {and: [
+      {
+        categories: {'@>': ['AA']},
+      },
+    ]}}, function(err, post) {
```

Comment:
I find this filter too complicated, would the following simpler version work too?

```js
Post.find({where: {categories: {'@>': ['AA']}}})
```

Reply:
Yes, @bajtos it is working fine too.